docs/src/main/mdoc/typeclasses/bimonad.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: docs
+title:  "Bimonad"
+section: "typeclasses"
+source: "core/src/main/scala/cats/Bimonad.scala"
+scaladoc: "#cats.Bimonad"
+---
+# Bimonad
+
+The `Bimonad` trait directly extends `Monad` and `Comonad` without introducing new behaviour. `Bimonad` is 
+different of other `Bi` typeclasses like `Bifunctor`, `Bifoldable` or `Bitraverse` where the prefix describes a
+`F[_, _]`. The `Bimonad` is a `F[_]` and could be better seen as a dual monad i.e. something that is both a `Monad` and 
```

Comment:
"dual monad" is another one that I worry is confusing. For one, `Comonad` is not a `Monad`. Actually it is the dual of `Monad`!
https://typelevel.org/cats/typeclasses/comonad.html

Reply:
The `DualMonad`\\`SelfDualMonad` thing i picked from old issues like #1297

I'll change it to:
```
The `Bimonad` is a `F[_]` thus the `Bi` prefix has a different meaning here: it's both a `Monad` and a `Comonad`.
```